cli/src/install/compose_db.rs

```diff
@@ -53,27 +53,32 @@ pub async fn install_compose_db(
     )
     .await?;
 
+    let network_name = if convert_network_identifier(&cfg.network.id) == "inmemory" {
```

Comment:
Rather than checking against strings, we can just case statement this

```
let network_id_for_model_list = if let NetworkIdentifier::InMemory = &cfg.network.id {
    NetworkIdentifier::Testnet
} else {
    cfg.network.id.clone()
};
let network_name = convert_network_identifier(network_id_for_model_list);
```